webrtc.html

```diff
@@ -3313,7 +3313,7 @@ <h2>Attributes</h2>
                 step labeled <em>create offer</em>.</p>
               </li>
               <li>
-                <p>If <var>connection</var> has any stopped "sendrecv" or
+                <p>If <var>connection</var> has any non-stopped "sendrecv" or
                 "recvonly" transceivers of type <var>mediaType</var>, jump to
```

Comment:
should it say "transceiver" (single)? Any, not part of this PR. I'll approve this one.